fixture_test.go

```diff
@@ -402,7 +402,9 @@ func (s *FixtureS) TestSetUpSuiteCheck(c *C) {
 			"fixture_test\\.go:[0-9]+:\n"+
 			"    c\\.Check\\(false, Equals, true\\)\n"+
 			"\\.+ obtained bool = false\n"+
-			"\\.+ expected bool = true\n\n")
+			"\\.+ expected bool = true\n"+
+			"\\.+ Values are different, diff:\n"+
+			"false \\!= true\n\n")
```

Comment:
This feels somewhat redundant (we have four lines saying true != false). We need to be a bit more strict about when the logic kicks in. If the goal is having typing, we might add additional logic inside the existing logValue function.

Reply:
Thats a very good point. I wonder what we should do here. We could simply omit this for all simple types. I.e. we do not show the difference for everything in `type.Basic. WDYT?

Reply:
As pointed out in a lot of cases this difference just adds noise. I updated the code to only do a diff if the type is complex and it wil also do very basic string diffs based on your suggestion to convert those into []string slices.